### DISCUSSION TITLE
Rotate Dashboard(s) at a frequency

### DISCUSSION THREAD

**Discussion by mweel1** [datalust/seq-tickets#1665] [Ideas]
It would be cool if we could get a play action on dashboard list, that would cycle through the dashboards at some frequency.

This way we could put it on TV displays and monitor dashboards appropriately. 

Great product!  Thank you!!

**Comment by liammclennan** [comment]
Thanks @mweel1. That is an interesting idea and I'm glad you like Seq. :)

Have you tried using a tab rotator browser plugin? We used one on the display at our NDC Oslo booth last week and it worked well.

**Reply by mweel1** [reply]
No, but that would work also.   Thx!
